.gitlab-ci.yml

```diff
@@ -191,38 +138,29 @@ after_script:
 
 build:base:
   <<: *build-template
-  variables:
-    COQ_EXTRA_CONF: "-native-compiler yes -coqide opt"
-    # coqdoc for stdlib, until we know how to build it from installed Coq
-    EXTRA_TARGET: "stdlib"
 
 # no coqide for 32bit: libgtk installation problems
 build:base+32bit:
   <<: *build-template
+  script:
+    - make voboot
+    - dune build coq.install
+    - dune build coqide-server.install
```

Comment:
Does dune fail when doing make world if gtk isn't installed?

Reply:
As of today only libraries can be optional, so yes it complains it cannot build `coqide` as `world` is an alias today for "all the packages". Note that this is not a fundamental problem as we could well indeed do as we do here and only build the relevant packages.

Soon, libraries and even packages will be allowed to be declared optional so indeed the error will be just a warning when building all the packages.